models/Image.js

```diff
@@ -8,7 +9,72 @@ var Image = new keystone.List('Image', {
   map: { name: 'description' }
 });
 
-var bucket = config['options']['gcs config']['bucket'];
+const bucket = config['options']['gcs config']['bucket'];
+
+const resizedOpts = [['desktop', 2000], ['tablet', 1200], ['mobile', 800], ['w400', 400], ['tiny', 150]];
+
+/**
+ *  Trigger `ResizeImage` Google Cloud Function.
+ *
+ *  @param {string} filename - uploaded file name
+ *  @returns {Promise} resolve with an array of message ids or reject with error
+ */
+function triggerResizeCloudFunction(filename) {
```

Comment:
minor. The function only publishes event rather than triggers CF directly so `publishResizeTasks() `would be better.